notmuch-addrlookup.c

```diff
@@ -217,11 +218,47 @@ print_contact_info_mutt (gpointer data, gpointer userdata)
   g_print ("%s\t%s\n", info->mail, info->name);
 }
 
+static void
+print_contact_info_aerc (gpointer data, gpointer userdata)
+{
+  GRegex *replace_re = g_regex_new ("\\\"?([^\\\"]+)\\\"?",
```

Comment:
This creates a new `GRegex` each time the function is called, but the object is never freed. We only need to create it once, and then it can be reused for printing each entry.

To do that, we can mark the variable as `static` (which uses the same storage for each call) and create it only the first time:

```suggestion
  static GRegex *replace_re = NULL;
  if (!replace_re)
    replace_re = g_regex_new ("\\\"?([^\\\"]+)\\\"?",
```

Reply:
makes sense! I didn't know it works that way.